# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv_headers/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv_headers/Cargo.toml b/third_party/rust/spirv/Cargo.toml
rename from third_party/rust/spirv_headers/Cargo.toml
rename to third_party/rust/spirv/Cargo.toml
--- a/third_party/rust/spirv_headers/Cargo.toml
+++ b/third_party/rust/spirv/Cargo.toml
@@ -7,33 +7,34 @@
 #
 # If you believe there's an error in this file please file an
 # issue against the rust-lang/cargo repository. If you're
 # editing this file be aware that the upstream Cargo.toml
 # will likely look very different (and much more reasonable)
 
 [package]
 edition = "2018"
-name = "spirv_headers"
-version = "1.5.0"
+name = "spirv"
+version = "0.2.0+1.5.4"
 authors = ["Lei Zhang <antiagainst@gmail.com>"]
 description = "Rust definition of SPIR-V structs and enums"
-documentation = "https://docs.rs/spirv_headers"
+documentation = "https://docs.rs/spirv"
 readme = "README.md"
 keywords = ["spirv", "definition", "struct", "enum"]
 license = "Apache-2.0"
 repository = "https://github.com/gfx-rs/rspirv"
 
 [lib]
 path = "lib.rs"
 [dependencies.bitflags]
 version = "1"
 
 [dependencies.num-traits]
 version = "0.2"
+default-features = false
 
 [dependencies.serde]
 version = "1"
 features = ["derive"]
 optional = true
 
 [features]
 deserialize = ["serde"]